extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -169,6 +171,20 @@ protected function renderImage( $data ) {
 		return $this->render( $templateName, $data );
 	}
 
+	protected function filterImageData( $data ) {
```

Comment:
I guess this should be a `private` method. Is there a reason for it to be protected? Could you move it below `render` functions as well?